eras/conway/impl/src/Cardano/Ledger/Conway/Governance/Procedures.hs

```diff
@@ -627,3 +668,32 @@ instance EraPParams era => EncCBOR (GovAction era) where
         Sum NewConstitution 5 !> E (encodeNullStrictMaybe encCBOR) gid !> To c
       InfoAction ->
         Sum InfoAction 6
+
+govActionParentL :: Lens' (GovAction era) (Maybe (GovActionId (EraCrypto era)))
```

Comment:
Same question about the lenses here. Should this lens be split into two? One preserving the type-level purpose and the other from any type-level purpose to the result?

Reply:
No, this kind of lenses should not exist. Create a getter function or a `SimpleGetter` that extracts it, that will be safe.

Reply:
Your suggestions have significantly simplified everything and we don't need this lens now. It was only needed in the complicated and misunderstood earlier implementation that changed `Ratify`.